metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala

```diff
@@ -51,17 +53,36 @@ object ClientCommands {
        |""".stripMargin
   )
 
-  val RunMain = Command(
-    "metals-main-run",
-    "run",
-    "Runs main method",
-    """|Array of strings of length 2 where
-       |- first array element is a build target identifier URI
-       |- second array element is the name of the class containing the main method 
-       |Example: `["mybuild://workspace/foo/?id=foo", "com.app.Main"]`
+  val StartDebugSession = Command(
+    "metals-debug-session-start",
+    "Start debug session",
+    s"""|Starts a debug session. The address of a new Debug Adapter can be obtained 
+        | by using the ${ServerCommands.StartDebugAdapter.id} metals server command
+        | with the same arguments as provided to this command.
+    """.stripMargin,
+    s"""|DebugSessionParameters object. It should be forwarded
+        |to the ${ServerCommands.StartDebugAdapter.id} command as is.
+        |
+        |Example:
+        |```json
+        |{
+        |  "targets": ["mybuild://workspace/foo/?id=foo"],
+        |   dataKind: "${b.DebugSessionParamsDataKind.SCALA_MAIN_CLASS}",
+        |   data: {
+        |      className: "com.foo.App"
+        |   }
+        |}```
     """.stripMargin
   )
 
+  val RefreshModel = Command(
+    "metals-model-refresh",
+    "Refresh model",
+    "Notifies the client that the model has been updated " +
```

Comment:
Use multiline string?

Reply:
Multiline strings impose a line break, which might not be desirable in this case here. I think these descriptions are always formatted as markdown however, so the line breaks won't affect the formatting of the output as long. Empty lines separate paragraphs however.